luigi/target.py

```diff
@@ -23,8 +23,15 @@
 import io
 import os
 import random
+import shutil
 import tempfile
 import logging
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
if luigi is dropping python 2.6 support  we can use ` from io import StringIO` only as this module is supported both in 2.7 and 3+